theme-utils.mjs

```diff
@@ -1099,7 +1099,17 @@ export async function executeCommand(command, logResponse) {
 				detached: true,
 			})
 		} else {
-			child = spawn(process.env.SHELL, ['-c', command], {
+			/*
+			* Determines the shell to execute the command.
+			* - Prioritizes using the user's default shell unless it's fish, a known problematic shell.
+			* - In this case, falls back to `/bin/bash` for better syntax compatibility.
+			*/
+			let shellPath = process.env.SHELL || '/bin/bash';
+			if (shellPath.includes('fish') && fs.existsSync('/bin/bash')) {
+				shellPath = '/bin/bash';
+			}
```

Comment:
I think it would be better to `WARN` about the problems that `fish` may cause instead of never allowing its use.

Reply:
The warning won't help out much, as when fish is my shell, running "bash" then running the themes commands still doesn't work. Changes like this PR are the only way I can get the script to deploy.